### ISSUE TITLE
About the dataset splitting way

### ISSUE THREAD

**Issue by huohuaqi** [Aftaab99/OfflineSignatureVerification#29]
In the paper [SigNet:.... Signature Verification], its author use the way that 'select 50 signer in total 55 signer' to split the train set and test set. 
However in this project code, you use the random select 13500*2 positive and negtive sample, and then 'train_test_split(data, test_size=0.15)'. By your way, it's more easy too much than the original method in the paper!!

So, in order to prove the effect of SigNet, I suggest that Reappearance  must be same！
